src/content/reference/manifest.mdx

```diff
@@ -60,6 +60,20 @@ Each image supports the following fields:
 - `args`: add build arguments, which are environment variables accessible only during the build process. Build arguments with a value containing a `$` sign are resolved to the environment variable value on the machine okteto is running on.
 - `export_cache`: image tag for exported cache when build.
 - `cache_from`: list of images to import cache from.
+- `secrets`: list of secrets exposed to the build.
+
+To expose a secret during the build of the service, you can define:
+
+```yaml
+build:
+  api:
+    context: api
+    secrets:
+      id1: path1
+      id2: path2
+```
+
+This will be equivalent run to the command: `okteto build api --secret id=id1,src=path1 --secret id=id2,src=path2`.
```

Comment:
```suggestion
This will mount the files path1 and paths1 as [mount secrets](https://github.com/moby/buildkit/blob/master/frontend/dockerfile/docs/reference.md#run---mounttypesecret)`.
```